data/glsk/glsk-document-cse/src/main/java/com/farao_community/farao/data/glsk/cse/CseGlskRegisteredResource.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.data.glsk.cse;
+
+import com.farao_community.farao.data.glsk.api.AbstractGlskRegisteredResource;
+import org.w3c.dom.Element;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
+ */
+public class CseGlskRegisteredResource extends AbstractGlskRegisteredResource {
+    private final Double initialFactor;
+
+    public CseGlskRegisteredResource(Element element) {
+        Objects.requireNonNull(element);
+        this.name = ((Element) element.getElementsByTagName("Name").item(0)).getAttribute("v");
+        this.mRID = this.name;
+        this.initialFactor = getContentAsDoubleOrNull(element, "Factor");
+        this.maximumCapacity = negativeIfNotNull(getContentAsDoubleOrNull(element, "Pmax"));
+        this.minimumCapacity = negativeIfNotNull(getContentAsDoubleOrNull(element, "Pmin"));
+    }
+
+    void setParticipationFactor(double participationFactor) {
+        this.participationFactor = participationFactor;
+    }
+
+    private Double getContentAsDoubleOrNull(Element baseElement, String tag) {
+        return baseElement.getElementsByTagName(tag).getLength() == 0 ? null :
+                Double.parseDouble(((Element) baseElement.getElementsByTagName(tag).item(0)).getAttribute("v"));
+    }
+
+    private Double negativeIfNotNull(Double value) {
```

Comment:
What is that for ?

Reply:
To make the code more easy to understand (compared to:
`getContentAsDoubleOrNull(element, "Pmax") != null ? - getContentAsDoubleOrNull(element, "Pmax) : null;`
)

Reply:
Maybe you can explicit in comments that initially in the document Pmin/Pmax seems to be negative so we take the absolute value to make algorithms easier. I just fear that if one day Pmin/Pmax turns to positive values it leads to some bugs.

Reply:
OK because it is UCTE convention for production